[ 2014-05-19 18:02:07.0647 18950/0x7fff7a41c310 agents/Watchdog/Main.cpp:538 ]: Options: { 'analytics_log_user' => 'roweandesign', 'cleanup_pidfiles' => 'L3Zhci9mb2xkZXJzL2JfL3gwXzJ3c25qNjJzODVkd3B4bDJsbDZ4MDAwMDBncC9UL3Bhc3Nlbmdlci1zdGFuZGFsb25lLjEwYmJlZTkvdGVtcF9kaXJfdG91Y2hlci5waWQ=', 'default_group' => 'staff', 'default_python' => 'python', 'default_ruby' => '/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/bin/ruby', 'default_user' => 'roweandesign', 'log_level' => '0', 'max_pool_size' => '6', 'passenger_root' => '/var/folders/b_/x0_2wsnj62s85dwpxl2ll6x00000gp/T/passenger-standalone.10bbee9/locations.ini', 'passenger_version' => '4.0.40', 'pool_idle_time' => '300', 'prestart_urls' => 'aHR0cDovLzAuMC4wLjA6MzAwMAA=', 'temp_dir' => '/tmp', 'union_station_gateway_address' => 'gateway.unionstationapp.com', 'union_station_gateway_port' => '443', 'user_switching' => 'false', 'web_server_passenger_version' => '4.0.40', 'web_server_pid' => '18926', 'web_server_type' => 'nginx', 'web_server_worker_gid' => '-1', 'web_server_worker_uid' => '-1' }
[ 2014-05-19 18:02:07.1842 18958/0x7fff7a41c310 agents/HelperAgent/Main.cpp:643 ]: PassengerHelperAgent online, listening at unix:/tmp/passenger.1.0.18926/generation-0/request
[ 2014-05-19 18:02:07.2294 18960/0x7fff7a41c310 agents/LoggingAgent/Main.cpp:321 ]: PassengerLoggingAgent online, listening at unix:/tmp/passenger.1.0.18926/generation-0/logging
[ 2014-05-19 18:02:07.2296 18950/0x7fff7a41c310 agents/Watchdog/Main.cpp:728 ]: All Phusion Passenger agents started!
App 6077 stdout: 
App 6077 stderr: env: python: No such file or directory
App 6077 stdout: 
PassengerHelperAgent(6183,0x10b1c6000) malloc: *** mach_vm_map(size=8388608) failed (error code=268435459)
PassengerHelperAgent(6183,0x10b1c6000) malloc: *** error: can't allocate region securely
PassengerHelperAgent(6183,0x10b1c6000) malloc: *** set a breakpoint in malloc_error_break to debug
[ 2014-05-20 12:31:07.5654 18958/0x10b1c6000 Pool2/Implementation.cpp:883 ]: Could not spawn process for group /Users/roweandesign/Sites/personal-website#default: An error occured while starting up the preloader.
     in 'void Passenger::ApplicationPool2::SmartSpawner::handleErrorResponse(Passenger::ApplicationPool2::SmartSpawner::StartupDetails &)' (SmartSpawner.h:455)
     in 'string Passenger::ApplicationPool2::SmartSpawner::negotiatePreloaderStartup(Passenger::ApplicationPool2::SmartSpawner::StartupDetails &)' (SmartSpawner.h:566)
     in 'void Passenger::ApplicationPool2::SmartSpawner::startPreloader()' (SmartSpawner.h:206)
     in 'virtual ProcessPtr Passenger::ApplicationPool2::SmartSpawner::spawn(const Passenger::ApplicationPool2::Options &)' (SmartSpawner.h:752)
     in 'void Passenger::ApplicationPool2::Group::spawnThreadRealMain(const SpawnerPtr &, const Passenger::ApplicationPool2::Options &, unsigned int)' (Implementation.cpp:804)

[ 2014-05-20 12:31:07.5661 18958/0x10b183000 agents/HelperAgent/RequestHandler.h:2210 ]: [Client 21] Cannot checkout session.
Error page:
cannot load such file -- bundler/setup (LoadError)
  /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:45:in `require'
  /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:45:in `require'
  /usr/local/Cellar/passenger/4.0.40/libexec/lib/phusion_passenger/loader_shared_helpers.rb:245:in `block in run_load_path_setup_code'
  /usr/local/Cellar/passenger/4.0.40/libexec/lib/phusion_passenger/loader_shared_helpers.rb:348:in `running_bundler'
  /usr/local/Cellar/passenger/4.0.40/libexec/lib/phusion_passenger/loader_shared_helpers.rb:243:in `run_load_path_setup_code'
  /usr/local/Cellar/passenger/4.0.40/libexec/helper-scripts/rack-preloader.rb:100:in `preload_app'
  /usr/local/Cellar/passenger/4.0.40/libexec/helper-scripts/rack-preloader.rb:158:in `<module:App>'
  /usr/local/Cellar/passenger/4.0.40/libexec/helper-scripts/rack-preloader.rb:29:in `<module:PhusionPassenger>'
  /usr/local/Cellar/passenger/4.0.40/libexec/helper-scripts/rack-preloader.rb:28:in `<main>'

